db/migrate/20211102150517_cap_changes_for_over_ordered_schools.rb

```diff
@@ -0,0 +1,26 @@
+class CapChangesForOverOrderedSchools < ActiveRecord::Migration[6.1]
+  def change
+    rename_table :allocation_changes, :cap_changes
+    rename_column :cap_changes, :prev_allocation, :prev_cap
+    rename_column :cap_changes, :new_allocation, :new_cap
+    change_column_null :cap_changes, :prev_cap, false, default: 0
+    change_column_null :cap_changes, :new_cap, false, default: 0
+    change_column_default :cap_changes, :prev_cap, from: nil, to: 0
+    change_column_default :cap_changes, :new_cap, from: nil, to: 0
+
+    restore_school_assigned_allocations
+  end
+
+private
+
+  def restore_school_assigned_allocations
+    School.includes(:cap_changes).find_each do |school|
```

Comment:
`School.joins(:allocation_changes)` should scope this to Schools with an associated AllocationChange.

Reply:
True. Changed!